R/manifest.R

```diff
@@ -328,6 +347,27 @@ update_data_flow_manifest <- function(asset_view,
       message(e)
     }
   )
+  
+  files <- lapply(synapse_manifests$dataset_id, function(x) {
```

Comment:
I could clean this up and put into its own function

Reply:
I think it would make sense to clean the duplicate code up and put it in a function